ebssurrogate/scripts/surrogate-bootstrap.sh

```diff
@@ -127,6 +131,7 @@ cat > "/mnt/etc/fstab" << EOF
 $(printf "${FMT}" "# DEVICE UUID" "MOUNTPOINT" "TYPE" "OPTIONS" "DUMP" "FSCK")
 $(findmnt -no SOURCE /mnt | xargs blkid -o export | awk -v FMT="${FMT}" '/^UUID=/ { printf(FMT, $0, "/", "ext4", "defaults,discard", "0", "1" ) }')
 $(findmnt -no SOURCE /mnt/boot/efi | xargs blkid -o export | awk -v FMT="${FMT}" '/^UUID=/ { printf(FMT, $0, "/boot/efi", "vfat", "umask=0077", "0", "1" ) }')
+$(findmnt -no SOURCE /mnt/data | xargs blkid -o export | awk -v FMT="${FMT}" '/^UUID=/ { printf(FMT, $0, "/mnt/data", "ext4", "defaults,discard", "0", "2" ) }')
```

Comment:
@Lakshmipathi I've built the AMI but unfortunately forgot to replace `/mnt/data` with `/data` in the fstab generation block here, so it took a while to properly rebuild it

The AMI ID is `ami-0e4637b9d37e044aa`, in ap-southeast-1.